Tests/AppTests/Common/Helpers/Controllers/Registration/ApproveHelper.swift

```diff
@@ -0,0 +1,61 @@
+import Vapor
+import Foundation
+@testable import App
+
+//swiftlint:disable superfluous_disable_command
+//swiftlint:disable force_try
+final class ApproveHelper {
+  
+  @discardableResult
+  static func store(places: [Place] = [], cities: [City] = []) throws -> [App.Event]? {
+    
+    var cityId = [Identifier]()
+    var placeId = [Identifier]()
+    var events = [App.Event]()
+    let randomRange = 1...Int.random(min: 10, max: 20)
+    let months = 24
+    
+    if cities.count == 0 {
```

Comment:
use `isEmpty` instead of `.count == 0` in all places where u use it please

Reply:
fixed